packages/clay-empty-state/src/index.tsx

```diff
@@ -42,15 +48,26 @@ const ClayEmptyState: React.FunctionComponent<IProps> = ({
 	return (
 		<div
 			className={classNames(className, 'c-empty-state', {
-				'c-empty-state-animation': hasImg,
+				'c-empty-state-animation': hasImg && !small,
+				'c-empty-state-animation-sm': hasImg && small,
 			})}
 			{...otherProps}
 		>
 			{hasImg && (
-				<div className="c-empty-state-image">
+				<div
+					className={classNames(
+						className,
+						{
+							'c-empty-state-image': !small,
+						},
+						{
+							'c-empty-state-image-sm': small,
+						}
```

Comment:
This is just a small change 😁 but in these cases, you can group everything into one object, it wouldn't be necessary to separate each condition into an object.

Reply:
You are right, I haven't seen that before 😄